README.md

```diff
@@ -67,6 +67,7 @@ We encourage the use of any libraries for everything except the search functiona
 1. Clone the candidate's branch
 1. Execute the following:
 
+        bin/bundle
```

Comment:
As a reviewer, I could not drop the database without first installing the new dependencies.